packages/exporter-trace-otlp-grpc/README.md

```diff
@@ -26,7 +26,7 @@ const { BasicTracerProvider, SimpleSpanProcessor } = require('@opentelemetry/sdk
 const { OTLPTraceExporter } =  require('@opentelemetry/exporter-trace-otlp-grpc');
 
 const collectorOptions = {
-  // url is optional and can be omitted - default is grpc://localhost:4317
+  // url is optional and can be omitted - default is localhost:4317
   url: 'grpc://<collector-hostname>:<port>',
```

Comment:
should we advise to set the url without the prococol to be consistent ?

Reply:
I think we can do that. To make it easier on the reader, we can also add a comment that says if they set the url without a scheme, the exporter will add "https" as the scheme.